src/server_manager/web_app/app.ts

```diff
@@ -1029,7 +1033,9 @@ export class App {
       this.appRoot.showNotification(this.appRoot.localize('saved'));
       serverView.accessKeyDataLimit = dataLimitToDisplayDataAmount(limit);
       this.refreshTransferStats(this.selectedServer, serverView);
-      this.surveys.presentDataLimitsEnabledSurvey();
+      // Don't display the feature collection disclaimer anymore.
+      serverView.showFeatureMetricsDisclaimer = false;
+      window.localStorage.setItem('dataLimits-feature-collection-consent', 'true');
```

Comment:
Then this would be `dataLimits-feature-collection-notification` or something similar.

Reply:
Done.